src/inmanta/resources.py

```diff
@@ -476,10 +476,15 @@ def get_managed(exporter: "export.Exporter", obj: "ManagedResource") -> bool:
     r"(?P<attr>[^=]+)=(?P<value>[^\]]+)\])(,v=(?P<version>[0-9]+))?$"
 )
 
+PARSE_RVID_REGEX = re.compile(
+    r"^(?P<id>(?P<type>(?P<ns>[\w-]+::)+(?P<class>[\w-]+))\[(?P<hostname>[^,]+),"
+    r"(?P<attr>[^=]+)=(?P<value>[^\]]+)\]),v=(?P<version>[0-9]+)$"
```

Comment:
This allows only for a single attribute, is this always the case?

Reply:
Yes. This is always the case.